packages/pancake-uikit/src/theme/colors.ts

```diff
@@ -32,6 +32,11 @@ export const lightColors: Colors = {
   card: "#FFFFFF",
   gradients: {
     bubblegum: "linear-gradient(139.73deg, #E6FDFF 0%, #F3EFFF 100%)",
+    cardBackground: {
```

Comment:
imo this should ba at the root like bubblegum (generic color)

Reply:
Sure,

```js
  gradients: {
    bubblegum: "linear-gradient(139.73deg, #313D5C 0%, #3D2A54 100%)",
    cardBackgroundDefault: "linear-gradient(166.77deg, #3B4155 0%, #3A3045 100%)",
    cardBackgroundBlue: "linear-gradient(180deg, #00707F 0%, #19778C 100%)",
    cardBackgroundViolet: "linear-gradient(180deg, #6C4999 0%, #6D4DB2 100%)",
  },
```

Or should we get bugs to come up with some thematic names?

Reply:
Bubblegum is what is was called in Figma.

Reply:
Updated this to remove nesting & reflect Figma color scheme